packages/web3/src/index.ts

```diff
@@ -321,6 +321,6 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 /**
  * This comment _supports3_ [Markdown](https://marked.js.org/)
  */
-import Web3 from './web3';
+import { Web3 } from './web3';
```

Comment:
Why not do both module export and named export instead of just module export?
Named export generally works better with IDE (auto-import and type hinting)